src/transformer.ts

```diff
@@ -53,6 +53,14 @@ export function transformerFactory<T extends TransformerNode>(
         return ts.visitEachChild(node, pathReplacer, context);
       }
 
+      /**
+       * e.g.
+       * - import('path').Type;
```

Comment:
It should be replaced with: `type Foo = import ('path').Foo`

Because: `import('path').Foo` similar to `import('path')`

Reply:
Sure! I updated it in the latest commit.